docs/index.md

```diff
@@ -125,18 +125,64 @@ Within your existing `didFinishLaunchingWithOptions` method, add the following t
 
 Beginning with firebase-ios-sdk v9+ (react-native-firebase v15+) you must tell CocoaPods to use frameworks.
 
-Open the file `./ios/Podfile` and add this line inside your targets:
+Open the file `./ios/Podfile` and add this line inside your targets (right after `'config = use_native_modules!'`):
 
-```
-use_frameworks!
+```ruby
+use_frameworks! :linkage => :static
 ```
 
-To use Static Frameworks on iOS, you also need to manually enable this for the project with the following global to the top of your `/ios/Podfile` file:
+To use Static Frameworks on iOS, you also need to manually enable this for the project with the following global to your `/ios/Podfile` file:
 
 ```ruby
+# right after `use_frameworks! :linkage => :static`
 $RNFirebaseAsStaticFramework = true
 ```
 
+For build setting, add below codes inside your targets (right after `'__apply_Xcode_12_5_M1_post_install_workaround(installer)'`)
+
+```ruby
+  installer.pods_project.targets.each do |target|
+    target.build_configurations.each do |config|
+      config.build_settings["GCC_WARN_INHIBIT_ALL_WARNINGS"] = "YES"
+    end
+  end
```

Comment:
This is a matter of taste and must not be suggested as a general configuration for anyone
In general warnings are useful, they just aren't in my build demo context

```suggestion
```